functions-app/README.md

```diff
@@ -0,0 +1,17 @@
+# Functions App
+
+For long-running functions
+
+## Disabling functions Locally
+
+1. You need an environment variable named `AzureWebJobs.<function-name>.Disabled` set to 'true'
+2. There is an example env file `disable-functions.env` that may be used:
+    * `env $(cat disable-functions.env | grep Azure | xargs ) yarn start`
+
+## Creating a new function
+
+From the `functions` directory, execute `func function new`.
```

Comment:
didn't know that was a command, `func new` is shorter and what i've always used

Reply:
The 'function' context is optional, so both forms are correct.